scripts/docker/debian.8/Dockerfile

```diff
@@ -41,6 +41,10 @@ RUN apt-get update && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+# liblldb is needed so deb package build does not throw missing library info errors
+RUN wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key|sudo apt-key add -
+RUN echo "deb http://llvm.org/apt/jessie/ llvm-toolchain-jessie-3.6 main" > /etc/apt/sources.list.d/llvm-toolchain.list
+RUN apt-get update && apt-get -qqy install liblldb-3.6
```

Comment:
Generally speaking it is a best practice to clean up the apt cache and remove /var/lib/apt/lists.  This helps keep the image size down.  It would also make this consistent with the previous apt-get logic in the Dockerfile.